tools/packaging/kernel/build-kernel.sh

```diff
@@ -96,6 +98,7 @@ Options:
 	-a <arch>   	: Arch target to build the kernel, such as aarch64/ppc64le/s390x/x86_64.
 	-b <type>    	: Enable optional config type.
 	-c <path>   	: Path to config file to build the kernel.
+	-D <vendor> 	: DPU/SmartNIC vendor, only nvidia.
```

Comment:
```suggestion
	-D <vendor> 	: DPU/SmartNIC vendor, only NVIDIA.
```

Reply:
>readonly GV_NVIDIA="nvidia"
> 	-g <vendor> 	: GPU vendor, intel or nvidia.

the const var and option `-g` use lower case, so here uses lower case should be fine.

Reply:
@l8huang I see that this PR is for adding kernel configs for NVIDIA Mellanox ConnectX network adaptor. While the PR looks good overall, the usage of NVIDIA here and in  `tools/packaging/kernel/configs/fragments/dpu/nvidia.conf` file name you added is slightly confusing, also due to the fact we have fragments present for NVIDIA gpu as well. 

Since the configs added in the fragment here have `Mellonox/MLX` in the name, can we rename this to Mellanox instead to reduce confusion. 

I am suggesting to replace 
`-D <vendor> 	: DPU/SmartNIC vendor, only NVIDIA.` => `-D <vendor> 	: DPU/SmartNIC vendor, only Mellanox.`


cc @zvonkok @fidencio